docs/01-Configuration.md

```diff
@@ -0,0 +1,322 @@
+# Configuration
+
+This document is meant to be an exhaustive reference of all TOML configuration fields available in Steward. Some of these fields are also used by the Orchestrator.
+
+## Default configuration
+
+You can see all available configuration fields with their default values by running
+
+```bash
+steward print-config
+```
+
+To generate a default config file, simply redirect stdout to a file:
+
+```bash
+# You must update these values to your own configuration
+steward print-config > config.toml
+```
+
+## Reference
+
+### Top-level table
```

Comment:
I'm confused by the use of the word "table" in this doc. In TOML there are only config key-value pairs and section names.

Reply:
TOML docs refer to hash tables/dictionaries as tables. https://toml.io/en/v1.0.0#table

I can just say top-level

Reply:
Weird, guess I was wrong on this one. Their spec would be the right way.